spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/filters/ServletTraceableRequest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.zuul.filters;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.boot.actuate.trace.http.TraceableRequest;
+import org.springframework.util.StringUtils;
+import org.springframework.web.util.UriUtils;
+
+/**
+ * @author Spencer Gibb
+ */	
+public class ServletTraceableRequest implements TraceableRequest {
+
+		private HttpServletRequest request;
+
+		ServletTraceableRequest(HttpServletRequest request) {
+			this.request = request;
+		}
+
+		@Override
+		public String getMethod() {
+			return request.getMethod();
+		}
+
+		@Override
+		public URI getUri() {
+			String queryString = this.request.getQueryString();
+			if (!StringUtils.hasText(queryString)) {
+				return URI.create(this.request.getRequestURL().toString());
```

Comment:
Please remove unnecessary use of `this.` throughout this class.

Reply:
Done